test/go-tests/test_importer_test.go

```diff
@@ -0,0 +1,163 @@
+package go_tests
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/stretchr/testify/require"
+	"io"
+	"mime/multipart"
+	"net/http"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+)
+
+const importerShipyard = `--- 
+apiVersion: "spec.keptn.sh/0.2.0"
+kind: "Shipyard"
+metadata:
+  name: "shipyard"
+spec:
+  stages:
+    - name: "dev"
+      sequences:
+        - name: "delivery"
+          tasks:
+            - name: "deployment"
+              properties:
+                deploymentstrategy: "user_managed"`
```

Comment:
```suggestion
            - name: "deployment"
```
do we really need the strategy? I think it's not useful for the test

Reply:
No, I removed it.